pkgs/misc/uboot/default.nix

```diff
@@ -178,6 +179,35 @@ in {
     '';
   };
 
+  ubootNanoPCT4 = buildUBoot rec {
```

Comment:
Non-mainline U-Boots live in distinct files. You couldn't have known as we removed the last one last year.

Look at 19.09 to see how it was setup

 - https://github.com/NixOS/nixpkgs/tree/nixos-19.09/pkgs/misc/uboot